.travis.yml

```diff
@@ -1,5 +1,24 @@
 language: elixir
 
+otp_release: 
+  - 23.0
+
+elixir:
+  - 1.10.4
+
+jobs:
+  include:
+    - elixir: 1.9.4
```

Comment:
I think you need to include 1.10.4/23.0 here as well or it won't actually run in travis